geth/common/types.go

```diff
@@ -385,14 +387,14 @@ type TestConfig struct {
 		HTTPPort    int
 		WSPort      int
 	}
-	Account1 struct {
-		Address  string
-		Password string
-	}
-	Account2 struct {
-		Address  string
-		Password string
-	}
+	Account1 account
+	Account2 account
+	Account3 account
+}
+
+type account struct {
```

Comment:
Could we move it above the first usage, that is above `TestConfig`?

Reply:
Got it.